glossary.md

```diff
@@ -122,7 +122,13 @@ More info at https://developer.apple.com/legacy/library/documentation/Darwin/Ref
 #### libc
 #### glibc
 #### libstdc++
+The [GNU implementation of the C++ STL](https://gcc.gnu.org/onlinedocs/libstdc++/).
+
+Note that on OS X, `libstdc++` does not support C++11 or C++14, and so we use `libc++` instead.  ([Read more](https://github.com/mapbox/cpp#standard-c-library).)
```

Comment:
:+1  Correct, because the version of libstdc++ bundled on is too old (and apple has not/does not plan to update it given they have moved to libc++). Theoretically you could build your own libstdc++ on OS X (libstdc++ is part of g++) but the recommended solution is to use libc++.